packages/gatsby-remark-images-contentful/src/utils/index.js

```diff
@@ -1,20 +1,34 @@
 const axios = require(`axios`)
 
-const getBase64Img = async url => {
-  const response = await axios({
-    method: `GET`,
-    responseType: `arraybuffer`,
-    url: `${url}`,
-  })
+const getBase64Img = async (url, reporter) => {
+  let base64Img
+  try {
+    const response = await axios({
+      method: `GET`,
+      responseType: `arraybuffer`,
+      url: `${url}`,
+    })
 
-  const base64Img = `data:${
-    response.headers[`content-type`]
-  };base64,${new Buffer(response.data).toString(`base64`)}`
+    base64Img = `data:${response.headers[`content-type`]};base64,${new Buffer(
+      response.data
+    ).toString(`base64`)}`
+  } catch (err) {
+    reporter.panic(`Failed downloading the base64 image for ${url}`, err)
+  }
 
   return base64Img
 }
 
-const buildResponsiveSizes = async ({ metadata, imageUrl, options = {} }) => {
+const buildResponsiveSizes = async (
+  { metadata, imageUrl, options = {} },
+  reporter
+) => {
+  // Remove search params from the image url
+  let formattedImgUrl = imageUrl
+  if (imageUrl.includes(`?`)) {
+    formattedImgUrl = imageUrl.split(`?`)[0]
+  }
```

Comment:
Could you use https://nodejs.org/api/url.html#url_url_parse_urlstring_parsequerystring_slashesdenotehost instead? It's more robust than doing a `?` check.

```suggestion
  const parsedImageUrl = parse(imageUrl);
  if (parsedImageUrl.search) {
    formattedImgUrl = `${parsedImageUrl.protocol}//${parsedImageUrl.host}${parsedImageUrl.pathname}`
  }
```
```

Reply:
Good catch. As web dev I guess we often forget that some things are in node already 😅 

Was about to write "luckily we can now use it"... but then looked up the version it got added to node.. Its v0.1.25... I worked to much in browser related libs :D